.github/workflows/docker.yaml

```diff
@@ -0,0 +1,80 @@
+name: Docker
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "v*.*.*"
+  pull_request:
+    branches:
+      - main
+  workflow_dispatch:
+    inputs:
+      tags:
+        description: "Tags to build and push"
+        required: true
+        type: string
+
+permissions:
+  contents: read
+  packages: write
```

Comment:
We need this to push to the docker registry, right?

Reply:
yup